### PR TITLE
code: document Eagle's Tower unused clouds tilemap

### PR DESCRIPTION
**Pull request by kemenaran**
Tilemap09 is actually the unused tilemap for animated Eagle's Tower clouds.

Zelda_LinksAwakening_Eagle_Tower

See [Eagle Tower Boss Fight](https://tcrf.net/The_Legend_of_Zelda:_Link%27s_Awakening_(Game_Boy)/Room_Design_Changes#Eagle_Tower_Boss_Fight) on The Cutting Room Floor.

### CODE REVIEW

**Comment by Rangi42** [comment]
The tilemap09.tilemap.encoded and tilemap09.attrmap.encoded files could also be renamed to `eagles_tower_clouds`, and the `tilemap07` files to `minimap`.

**Comment by kemenaran** [comment]
Indeed, good catch. Follow-up in #439.

File: src/code/world_handler.asm [hunk 1]
```diff
@@ -107,7 +109,7 @@ jr_001_43DC::
     ld   a, $0F                                   ; $43E7: $3E $0F
 .colorDungeonIndexEnd
 
-    ; Lookup the minimap layout for the dungeon in a
```
Comment (Xkeeper0) [inline src/code/world_handler.asm:110]: Was this intentional? That comment says "in a" as in "in the register a", not as a sentence fragment...

Comment (kemenaran) [reply src/code/world_handler.asm:110]: After trying to parse this sentence three times, that is also the conclusion I reached  😅

It wasn't that clear, though, and the register name wasn't absolutely needed, so I simplified the sentence.

**Pull request merged**